pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -2560,6 +2560,21 @@ func (gce *GCECloud) GetAutoLabelsForPD(name string, zone string) (map[string]st
 	return labels, nil
 }
 
+// TestDisk checks that a disk has given type. It should be used only for
+// testing!
+func (gce *GCECloud) TestDisk(diskName, volumeType string) error {
```

Comment:
I made `GCECloud.GetDiskByNameUnknownZone` and `GCEDisk` public and moved `TestDisk ` to the e2e test itself.